.env.example

```diff
@@ -1,31 +0,0 @@
-###########################
-#   Example `.env` file   #
-###########################
-
-# API key for the Google Civic API
-# See https://github.com/ProgramEquity/amplify/blob/main/.github/CONTRIBUTING.md#using-the-google-civic-information-api-locally
-CIVIC_API_KEY=
-
-# Production environment API key for the Lob API
-# This value can be found in Heroku application config var of the same name
-LOB_API_KEY=
-
-# Test environment API key for the Lob API
-# This is only required for running the integration tests successfully!
-# This value can be found in Heroku application config var of the same name
-TEST_LOB_API_KEY=
-
-# Auth0 authentication parameters
-# For local development, you can just use these literal nonsense values for now
-SERVER_PORT=8080
-CLIENT_ORIGIN_URL=http://localhost:8080
-AUTH0_AUDIENCE=your_Auth0_identifier_value
-AUTH0_DOMAIN=your_Auth0_domain
-
-# Stripe
-STRIPE_SECRET_KEY=sk_test_51IravfFqipIA40A3FOW6EzlXlJiXjL9V0FXKfb9n7cxh25Ww9QMA9aWwCzTSQscBOQFcB7s1TI6UCtW1JG83Hz1z000Sg2vSIr
-
-# To run in single-campaign mode, set the following environment vars
-VUE_APP_CAMPAIGN_MODE = "single" # or "default" for normaal operation
-VUE_APP_FEATURED_CAMPAIGN = "5" # campaign's id from Postgres
-VUE_APP_LETTER_TEMPLATE = "" # the template associated with the featured campaign
```

Comment:
Hey @imolina212 ! Looks like you accidently committed the deletion of the `.env.example` file. Please revert the file. Everything else looks good to me. :)

Reply:
Looks good to me also!